src/components/slide/slide.js

```diff
@@ -92,6 +89,7 @@ export default function Slide({
   padding,
   textColor,
   template,
+  transition: slideTransition = {},
```

Comment:
Should this be `defaultTransition` instead of `{}`? I see the import for `defaultTransition`, but don't see it being used - but I'd assume that'd go here?

Reply:
Oh good catch, Slide already gets `defaultTransition` from deck. I can remove that import. But if we made it `defaultTransition` it would just assign the same thing to itself.